modules/ldap-syncing-rfc2307.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-syncing-rfc2307_{context}"]
+= Syncing groups using the RFC 2307 schema
```

Comment:
I think these examples need to be set up more like procedure modules. Since you need to first create the config file, and then sync it, etc.

Reply:
Made the topics into procedures. Might be too much info for a procedure. But, it seems necessary to run the procedure.

Reply:
Outdated